lib/embulk/input/twitter_ads_analytics.rb

```diff
@@ -86,6 +86,7 @@ def self.guess(config)
         columns += [
           {name: "line_item_id", type: "string"},
           {name: "line_item_name", type: "string"},
+          {name: "campaign_id_in_line_item", type: "string"},
```

Comment:
```suggestion
          {name: "campaign_id", type: "string"},
```

just `campaign_id` will be fine.